noxfile.py

```diff
@@ -145,10 +182,24 @@ def _get_pypi_package_data(package_name):
 
 def _get_urls(data, version):
     return list(
-        r["url"] for r in data["releases"][version] if _contains(r["url"], ("cp37",))
+        r["url"] for r in data["releases"][version] if _contains(r["url"], ("cp310",))
     )
 
 
+def download_url(root, value):
+    with url_lib.urlopen(value["url"]) as response:
+        data = response.read()
+        if hashlib.sha256(data).hexdigest() == value["hash"]:
+            print("Download: ", value["url"])
+            with zipfile.ZipFile(io.BytesIO(data), "r") as wheel:
+                for zip_info in wheel.infolist():
+                    # Ignore dist info since we are merging multiple wheels
+                    if ".dist-info/" in zip_info.filename:
+                        continue
+                    print("\t" + zip_info.filename)
+                    wheel.extract(zip_info.filename, root)
```

Comment:
I don't see any error condition if the hash check fails, so we should make sure that some exception or error message an immediate exit occurs so we catch the problem.

We are also installing only a single wheel, so we should keep the `.dist-info` directory. 
```suggestion
    with url_lib.urlopen(value["url"]) as response:
        data = response.read()
    if hashlib.sha256(data).hexdigest() != value["hash"]:
        # XXX error condition that should abort everything
    print("Download: ", value["url"])
    with zipfile.ZipFile(io.BytesIO(data), "r") as wheel:
        for zip_info in wheel.infolist():
            print("\t", zip_info.filename)
            wheel.extract(zip_info.filename, root)
```